lib/core/CStaticThreadPool.cc

```diff
@@ -21,11 +22,18 @@ std::size_t computeSize(std::size_t hint) {
 }
 }
 
-CStaticThreadPool::CStaticThreadPool(std::size_t size)
-    : m_Busy{false}, m_Cursor{0}, m_TaskQueues{computeSize(size)} {
-    m_NumberThreadsInUse.store(m_TaskQueues.size());
-    m_Pool.reserve(m_TaskQueues.size());
-    for (std::size_t id = 0; id < m_TaskQueues.size(); ++id) {
+CStaticThreadPool::CStaticThreadPool(std::size_t size, std::size_t queueCapacity)
+    : m_Busy{false}, m_Cursor{0} {
+
+    std::size_t poolSize = computeSize(size);
```

Comment:
Nit coding conventions:
```suggestion
    std::size_t poolSize{computeSize(size)};
```